camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java

```diff
@@ -137,12 +139,13 @@ public void before() throws IOException, NoSuchFieldException, IllegalAccessExce
   public void after() throws IOException {
     // Delete all camus data
     folder.delete();
+    _mocks.clear();
```

Comment:
Or the _mocks shouldn't be an instance variable.  Since we are not sharing mocks between different tests, it probably should be method level variable.